android/src/main/java/com/thebluealliance/androidclient/activities/settings/NotificationSettingsActivity.java

```diff
@@ -51,18 +52,19 @@ public void onCreate(Bundle savedInstanceState) {
             notificationTone.setEnabled(currentlyEnabled);
             notificationVibrate.setEnabled(currentlyEnabled);
 
-            enableNotifications.setOnPreferenceChangeListener(new Preference.OnPreferenceChangeListener() {
-                @Override
-                public boolean onPreferenceChange(Preference preference, Object newValue) {
-                    boolean value = (Boolean) newValue;
-                    notificationTone.setEnabled(value);
-                    notificationVibrate.setEnabled(value);
-                    if (Build.VERSION.SDK_INT > Build.VERSION_CODES.KITKAT) {
+            enableNotifications.setOnPreferenceChangeListener((preference, newValue) -> {
+                boolean value = (Boolean) newValue;
+                notificationTone.setEnabled(value);
+                notificationVibrate.setEnabled(value);
+                if (Build.VERSION.SDK_INT > Build.VERSION_CODES.KITKAT) {
```

Comment:
Related: I think it's semantically nicer if the version checks are >= LOLLIPOP instead of > KITKAT. Maybe change that as long as we're modifying this file?